web-api/src/persistence/postgres/cases/generateDocketNumber.ts

```diff
@@ -50,32 +47,5 @@ export const generateDocketNumber = async ({
     ? formatDateString(receivedAt, FORMATS.YEAR)
     : formatNow(FORMATS.YEAR);
 
-  let attempt = 0;
```

Comment:
If we go the mutex route, we guarantee only one process is generating a docket number at a time.